client/package.json

```diff
@@ -1,56 +1,69 @@
 {
   "name": "therapy-dog",
   "version": "0.0.0",
-  "description": "Small description for therapy-dog goes here",
   "private": true,
+  "description": "Small description for therapy-dog goes here",
+  "repository": "",
+  "license": "MIT",
+  "author": "",
   "directories": {
     "doc": "doc",
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build",
-    "start": "ember server",
-    "test": "ember test"
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint:hbs": "ember-template-lint .",
+    "lint:js": "eslint .",
+    "start": "ember serve",
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test"
+  },
+  "devDependencies": {
+    "@ember/jquery": "^1.1.0",
+    "@ember/optional-features": "^1.3.0",
+    "@ember/test-helpers": "^1.7.2",
+    "@glimmer/component": "^1.0.1",
+    "@glimmer/tracking": "^1.0.0",
+    "babel-eslint": "^10.1.0",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-auto-import": "^1.6.0",
+    "ember-cli": "~3.20.0",
+    "ember-cli-app-version": "^3.2.0",
+    "ember-cli-babel": "^7.21.0",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-htmlbars": "^5.2.0",
+    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-uglify": "^3.0.0",
+    "ember-data": "~3.20.0",
+    "ember-export-application-global": "^2.0.1",
+    "ember-fetch": "^8.0.1",
+    "ember-load-initializers": "^2.1.1",
+    "ember-maybe-import-regenerator": "^0.1.6",
+    "ember-qunit": "^4.6.0",
+    "ember-resolver": "^8.0.0",
+    "ember-source": "~3.20.2",
+    "ember-template-lint": "^2.9.1",
+    "ember-welcome-page": "^4.0.0",
+    "eslint": "^7.5.0",
+    "eslint-plugin-ember": "^8.9.1",
+    "eslint-plugin-node": "^11.1.0",
+    "glob": "^7.1.6",
+    "http-proxy": "^1.18.1",
+    "jshint": "^2.12.0",
+    "loader.js": "^4.7.0",
+    "morgan": "^1.10.0",
+    "npm-run-all": "^4.1.5",
+    "qunit-dom": "^1.2.0"
   },
-  "repository": "",
   "engines": {
-    "node": ">= 0.12.0"
+    "node": "10.* || >= 12"
```

Comment:
If i'm reading this correctly its allowing node 10 or 12+? Are we actually checking that 10 works?

Reply:
No. That got auto updated. I can switch it to just >= 12